doc/files/files.html

```diff
@@ -758,7 +758,29 @@
 								</p></li><li class="listitem"><p>
 									<code class="literal">no</code>: File caching disabled.
 								</p></li></ul></div><p>
-							(Default:<code class="literal">no</code>).
+							(Default: <code class="literal">public</code>
+							for the following card drivers
+							<code class="literal">akis</code>,
+							<code class="literal">atrust-acos</code>,
+							<code class="literal">belpic</code>,
+							<code class="literal">cac1</code>,
+							<code class="literal">cac</code>,
+							<code class="literal">coolkey</code>,
+							<code class="literal">dnie</code>,
+							<code class="literal">edo</code>,
+							<code class="literal">esteid2018</code>,
+							<code class="literal">flex</code>,
+							<code class="literal">cyberflex</code>,
+							<code class="literal">gemsafeV1</code>,
+							<code class="literal">idprime</code>,
+							<code class="literal">itacns</code>,
+							<code class="literal">jpki</code>,
+							<code class="literal">MaskTech</code>,
+							<code class="literal">mcrd</code>,
+							<code class="literal">npa</code>,
+							<code class="literal">nqapplet</code>,
+							<code class="literal">PIV-II</code>,
```

Comment:
the html file still contains the piv driver in the list, but it is not in the actual list in the code nor in the xml. Can you update the HTML?

Reply:
done